pyrightconfig.json

```
@@ -2,12 +2,38 @@
+    "."
+  ],
+
   "exclude": [
     ".venv/**",
     "**/__pycache__",
     "**/migrations"
+  ],
+
+  "reportMissingImports": false,
+  "reportMissingModuleSource": false,
+  "reportMissingTypeStubs": false,
```

Biome (1.9.4)

[error] The key reportMissingTypeStubs was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)